Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1588,6 +1588,7 @@ def tableToMarkdown(name, t, headers=None, headerTransform=None, removeNull=Fals
         headers = list(t[0].keys())
         headers.sort()
 
+    headers = headers[:]
```

Comment:
to avoid multiple list duplications, I think this will have a better performance:
```python
if removeNull:
    headers_aux = headers[:]
    for header in headers:
            if all(obj.get(header) in ('', None, [], {}) for obj in t):
                headers_aux.remove(header)
    headers = headers_aux
```